modules/http/http-worker.c

```diff
@@ -246,51 +246,108 @@ _add_message_to_batch(HTTPDestinationWorker *self, LogMessage *msg)
     }
 }
 
+static gboolean
+find_http_code_in_list(glong http_code, glong list[])
```

Comment:
please use a _ prefix to indicate this is a private function.

Reply:
Fixed.